start

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-node chesster.js $1
+node chesster.js $1 >> stdout_log 2>> stderr_log
```

Comment:
I think I'd prefer to have a single log for all of it, something like

```2&>1 >> output.log```

I don't want to have to remember which log to look at for which information.